src/lib/components/Tools/DrawTool.svelte

```diff
@@ -2,42 +2,35 @@
 	import MenuButton from '$lib/components/Navbar/MenuButton.svelte';
 	import Icons from '$lib/icons/MenuIcons.json';
 	import { toolState, canvasTouched, socket, user } from '$lib/stores/stateStore';
-	import { onMount } from 'svelte';
 	import { writable } from 'svelte/store';
-	import { ToolState } from '$lib/types';
+	import { ToolState, type CommandId } from '$lib/types';
 
 	const STROKE_WIDTH = 7;
 	let currentCommandId = writable<number | null>(null);
 
-	onMount(() => {
-		$socket.on('startDrawSuccess', (data) => {
-			if (data.username !== $user.name) return;
-			$currentCommandId = data.commandId;
-		});
-	});
-
 	user.subscribe(startDraw);
 	user.subscribe(doDraw);
 	canvasTouched.subscribe(stopDraw);
 
 	function startDraw() {
 		if (!$canvasTouched || $toolState !== ToolState.draw || $currentCommandId !== null) return;
-		$socket.emit('startDraw', {
-			placement: { x: 0, y: 0 }, // TODO: Mads, remove placement from backend. Rename path->coords.
-			path: $user.position,
-			stroke: $user.drawColor,
-			fill: 'transparent',
-			strokeWidth: STROKE_WIDTH,
-			username: $user.name
-		});
+		$socket.emit(
+			'startDraw',
+			{
+				coordinate: $user.position,
```

Comment:
Jeg har skiftet mening. Placement giver mere mening end coordinate (hvilket iøvrigt nok burde være coordinates).

Coordinates er nemlig typen, placement er variablen. Hvis du er villig til det, synes jeg den skal hedde placement alligevel
```suggestion
				placement: $user.position,
```

Reply:
Det er også det de hedder i serveren